samples/examples/huggingface/huggingface.md

```diff
@@ -108,381 +107,395 @@ To allow Alibi-Explain to more easily explain the sentiment we will need:
-                ]
-              },
-              {
-                "name": "sentiment-input-transform",
-                "inputs": [
-                  "whisper.outputs"
-                ]
-              },
-              {
-                "name": "whisper"
-              }
-            ],
-            "output": {
-              "steps": [
-                "sentiment.outputs",
-                "whisper.outputs"
-              ]
-            },
-            "kubernetesMeta": {}
-          },
-          "state": {
-            "pipelineVersion": 1,
-            "status": "PipelineReady",
-            "reason": "created pipeline",
-            "lastChangeTimestamp": "2022-11-18T11:49:27.793654254Z"
-          }
-        }
-      ]
-    }
+```json
+{
+  "pipelineName": "speech-to-sentiment",
+  "versions": [
+    {
+      "pipeline": {
+        "name": "speech-to-sentiment",
+        "uid": "cdrn2htjr36s73eg9omg",
+        "version": 1,
+        "steps": [
+          {
+            "name": "sentiment",
+            "inputs": [
+              "whisper.outputs"
+            ],
+            "tensorMap": {
+              "whisper.outputs.output": "args"
+            }
+          },
+          {
+            "name": "sentiment-explainer",
+            "inputs": [
+              "sentiment-input-transform.outputs"
+            ]
+          },
+          {
+            "name": "sentiment-input-transform",
+            "inputs": [
+              "whisper.outputs"
+            ]
+          },
+          {
+            "name": "whisper"
+          }
+        ],
+        "output": {
+          "steps": [
+            "sentiment.outputs",
+            "whisper.outputs"
+          ]
+        },
+        "kubernetesMeta": {}
+      },
+      "state": {
+        "pipelineVersion": 1,
+        "status": "PipelineReady",
+        "reason": "created pipeline",
+        "lastChangeTimestamp": "2022-11-18T11:49:27.793654254Z"
+      }
+    }
+  ]
+}
 
+```
 
 ### Test
 
-
 ```python
 camera = CameraStream(constraints={'audio': True,'video':False})
 recorder = AudioRecorder(stream=camera)
 recorder
 ```
 
+```
+AudioRecorder(audio=Audio(value=b'', format='webm'), stream=CameraStream(constraints={'audio': True, 'video': …
 
-    AudioRecorder(audio=Audio(value=b'', format='webm'), stream=CameraStream(constraints={'audio': True, 'video': …
-
-
+```
 
 ```python
 infer("speech-to-sentiment.pipeline")
 ```
 
-    cdrn3iits0ps73bgtaeg
-    {"text": " Cambridge is wonderful and beautiful."}
-    {"label": "POSITIVE", "score": 0.9998691082000732}
+```
+cdrn3iits0ps73bgtaeg
+{"text": " Cambridge is wonderful and beautiful."}
+{"label": "POSITIVE", "score": 0.9998691082000732}
 
+```
 
 We will wait for the explanation which is run asynchronously to the functional output from the Pipeline above.
 
-
 ```python
 while True:
     base64Res = !seldon pipeline inspect speech-to-sentiment.sentiment-explainer.outputs --format json \
```

Comment:
:information_source: This is shelling out, but the block is mostly Python so I think it's better to leave this as it is rather than trying to be too clever and risking false positives based on `!` characters in the middle of things